src/app/progress/engrams.ts

```diff
@@ -28,46 +28,32 @@ const engrams = {
     cap: PowerCap.Powerful,
     bonus: 3,
   },
-  // Powerful Legacy
-  2246571627: {
-    cap: PowerCap.Powerful,
-    bonus: 3,
-  },
   // Tier 2
   3114385606: {
     cap: PowerCap.Powerful,
     bonus: 4,
   },
+  // Rose (from Competitive)
+  882778888: {
+    cap: PowerCap.Powerful,
+    bonus: 4,
+  },
```

Comment:
I went off the report in the linked issue but it'd be good if someone could double check this. I reached the powerful cap already though, and I'm not sure I can ask someone to subject themselves to this mode 😄

Reply:
I'm certainly not going to do it.